definitions/features/foreman_proxy.rb

```diff
@@ -34,16 +34,17 @@ def find_http_error_msg(array_output, curl_http_status)
         next unless str.include?('HTTP')
         http_line = str
       end
+      return http_line if http_line.empty?
       http_line.split(curl_http_status.to_s).last.strip
     end
   end
 
   def run_dhcp_curl
     curl_resp = execute(dhcp_curl_cmd)
-    array_output = curl_resp.split(/\r\n/)
+    array_output = curl_resp.split(/\r?\n/)
     result_array = array_output.last.split(/\n/)
```

Comment:
I don't understand this code: why are we doing split on `/\n/` for something that was just split based on that on the previous line. 

```ruby
"one\ntwo\nthree".split(/\r?\n/)
# => ["one", "two", "three"]
```
In which case the `array_output.last` can actually contain `\n`?

Reply:
yes, I will modify this.

Reply:
@iNecas ,
Corrected the code. Ready for review.